### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rishabhrathore055**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,13 +68,26 @@ Base template has been designed and developed by Themefisher. For a preview, che
 The code has been restructured, unnecessary code has been removed, comments were added wherever required, proper indentation was provided, and simple names were used which makes it a contributor-friendly codebase. So, hope on and contribute now!
 
 ## Feedback/Suggestions and Issues
+
 Feedback/Suggestions are always welcome and do mention any issues found. They can be provided through:
-1)Issues section of this repository
-2)[https://github.com/sugarlabs/www-sugarlabs/issues/63](https://github.com/sugarlabs/www-sugarlabs/issues/63)
+
+1. Issues section of this repository
+2. [https://github.com/sugarlabs/www-sugarlabs/issues/63](https://github.com/sugarlabs/www-sugarlabs/issues/63)
```
Comment (chimosky) [inline README.md:75]: You can change the hyperlink text to "Issue #63" and this removes the need for the hyperlink.

**Review by chimosky — CHANGES_REQUESTED** [review]

**Comment by chimosky** [comment]
Reviewed, thanks @rishabhrathore055.

You might want to check out our [contributing guide](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#contributing) for future contributions.

File: README.md (hunk 1 above)
Comment (rishabhrathore055) [reply README.md:75]: @chimosky  But it closed I guess 
So could explain what i have to so now

Comment (chimosky) [reply README.md:75]: Thanks for pointing that out, just noticed and there seems to be two sections of `Feedback/Suggestions and Issues`, you can remove one - preferably the second -.

Comment (rishabhrathore055) [reply README.md:73]: @chimosky Now Check it once

Comment (chimosky) [reply README.md:73]: The section includes the items listed.

Comment (rishabhrathore055) [reply README.md]: Now?

**Comment by chimosky** [comment]
Looks good, thanks @rishabhrathore055.

**Comment by rishabhrathore055** [comment]
@chimosky  Is Sugar lab participating in Gsoc 2022?

**Comment by chimosky** [comment]
The decision for that hasn't been made yet but we'll be sure to make announcements on our [devel list](https://lists.sugarlabs.org/listinfo/sugar-devel) when it's been made so you can subscribe to it.

**Comment by rishabhrathore055** [comment]
Thanks For the information

**Pull request merged**